.github/workflows/on-push.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         with:
           path: ./node_modules
           key: ${{ runner.os }}-${{hashFiles('**/package-lock.json')}}-node_modules
-      - run: npm install
+      - run: npm ci --no-optional
```

Comment:
We need this flag because we're adding `@atlassiansox/analytics-node-client` - see `package.json` below, which is a private package to P.A.C. and GitHub actions falls back to the public npmregistry, which fails our build.